test/test_compliance_kaldi.py

```diff
@@ -46,7 +46,10 @@ def first_sample_of_frame(frame, window_size, window_shift, snip_edges):
             window[f, s] = wave[s_in_wave]
 
 
-class Test_Kaldi(unittest.TestCase):
+@common_utils.skipIfNoSoxBackend
+class Test_Kaldi(common_utils.TorchaudioTestCase):
+    backend = 'sox'
```

Comment:
nit: only one the method in tests below needs sox, no?

Reply:
`load_wav` yields different value between sox backend and soundfile backend so this test does not work for soundfile backend.